source/features/next-scheduled-github-action.tsx

```diff
@@ -0,0 +1,85 @@
+import './next-scheduled-github-action.css';
+import React from 'dom-chef';
+import cache from 'webext-storage-cache';
+import select from 'select-dom';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+import {parseCron} from '@cheap-glitch/mi-cron';
+
+import features from '.';
+import * as api from '../github-helpers/api';
+import {getRepoURL, getRepoGQL} from '../github-helpers';
+
+const getActionsSchedules = cache.function(async (): Promise<{[index: string]: string} | false> => {
+	const {repository: {object: {entries: actions}}} = await api.v4(
+		`repository(${getRepoGQL()}) {
+			object(expression: "HEAD:.github/workflows") {
+				... on Tree {
+					entries {
+						object { ... on Blob { text } }
+					}
+				}
+			}
+		}
+	`);
+	if (!actions) {
+		return false;
+	}
+
+	const schedules: {[index: string]: string} = {};
+	for (const action of actions) {
+		const actionYaml = action.object.text;
+		const name = /^name:\s+['"]?(.+)['"]?/m.exec(actionYaml);
+		const cron = /^\s*-\scron:\s+['"](.+)['"]/m.exec(actionYaml);
+
+		if (name && cron) {
+			schedules[name[1]] = cron[1];
+		}
+	}
+
+	return schedules;
+}, {
+	maxAge: {days: 1},
+	staleWhileRevalidate: {days: 10},
+	cacheKey: () => __filebasename + ':' + getRepoURL()
+});
+
+async function init(): Promise<false | void> {
+	const actionsSidebar = await elementReady('.hx_actions-sidebar');
+	if (!actionsSidebar) {
+		return false;
+	}
+
+	const actionsSchedules = await getActionsSchedules();
+	if (!actionsSchedules) {
+		return false;
+	}
+
+	for (const actionListItem of select.all('li:not(:first-child) > a', actionsSidebar)) {
+		const actionName = actionListItem.textContent!.trim();
+		if (!(actionName in actionsSchedules)) {
+			continue;
+		}
+
+		const nextTime = parseCron.nextDate(actionsSchedules[actionName]);
+		if (nextTime) {
+			actionListItem.append(
+				<span className="rgh-github-action-next-time">
```

Comment:
```suggestion
				<em>
```

So you can drop the CSS